classes/PodsUI.php

```diff
@@ -4711,7 +4711,7 @@ public function screen_meta() {
 									if ( true === $this->pagination ) {
 										?>
 										<input type="text" class="screen-per-page" name="wp_screen_options[value]" id="<?php echo esc_attr( $this->unique_identifier ); ?>_per_page" maxlength="3" value="20">
-										<label for="<?php echo esc_attr( $this->unique_identifier ); ?>_per_page"><?php esc_html_e( sprintf( __( '%s per page', 'pods' ), $this->items ) ); ?></label>
+										<label for="<?php echo esc_attr( $this->unique_identifier ); ?>_per_page"><?php printf( esc_html__( '%s per page', 'pods' ), $this->items ); ?></label>
```

Comment:
We want to escape `$this->items` too but we didn't need to use `esc_html_e()` here like before since that's a translating function so we can use `echo esc_html()`

```suggestion
										<label for="<?php echo esc_attr( $this->unique_identifier ); ?>_per_page"><?php echo esc_html( sprintf( __( '%s per page', 'pods' ), $this->items ) ); ?></label>
```